packages/sanity/src/structure/panes/documentList/sheetList/DocumentSheetListPane.tsx

```diff
@@ -265,65 +275,64 @@ function DocumentSheetListPaneInner(
       return !isSomeOperationsDisabled
     }
     return false
-  }, [rowOperations, table.options.meta?.patchDocument])
+  }, [isLoading, rowOperations, table.options.meta?.patchDocument])
 
   const renderContent = () => {
-    if (!isReady) {
-      return <LoadingPane paneKey={paneProps.paneKey} />
-    }
+    if (!isReady)
+      return (
+        <Flex style={{height: '100%'}}>
+          <LoadingPane paneKey={paneProps.paneKey} />
+        </Flex>
+      )
 
     return (
-      <React.Fragment>
-        <TableActionsWrapper
-          direction="row"
-          align="center"
-          paddingY={3}
-          paddingX={1}
-          justify="space-between"
-        >
-          <Flex direction="row" align="center">
-            <DocumentSheetListFilter />
-            <Text size={0} muted>
-              {t('row-count.label', {
-                totalRows,
-                itemPlural: `item${totalRows === 1 ? '' : 's'}`,
-              })}
-            </Text>
-          </Flex>
-          <ColumnsControl table={table} />
-        </TableActionsWrapper>
-        <TableContainer>
-          <DocumentSheetListProvider table={table}>
-            <Table>
-              <thead>
-                {table.getHeaderGroups().map((headerGroup) => (
-                  <Box as="tr" key={headerGroup.id}>
-                    {headerGroup.headers.map((header) => (
-                      <DocumentSheetListHeader
-                        key={header.id}
-                        header={header}
-                        headerGroup={headerGroup}
-                        table={table}
-                      />
-                    ))}
-                  </Box>
-                ))}
-              </thead>
-              <tbody>{table.getRowModel().rows.map(renderRow)}</tbody>
-            </Table>
-          </DocumentSheetListProvider>
-          <DocumentSheetActions table={table} schemaType={documentSchemaType} />
-        </TableContainer>
-        <Flex justify={'flex-end'} padding={3} gap={4} paddingY={5}>
-          <DocumentSheetListPaginator table={table} />
-        </Flex>
-      </React.Fragment>
+      <Table>
```

Comment:
Reordering of the content behind the loader, so that it's only the table, and not also the filter bar, that is hidden until loaded